ypathgen/pathgen.go

```diff
@@ -262,6 +266,21 @@ func (cg *GenConfig) GeneratePathCode(yangFiles, includePaths []string) (*Genera
 		errs = util.AppendErrs(errs, es)
 	}
 
+	namer := func(dir *ygen.Directory, dirs map[string]*ygen.Directory) string {
+		if !cg.SplitByModule {
+			return cg.PackageName
+		}
+		if ygen.IsFakeRoot(dir.Entry) {
+			return cg.FakeRootPackageName
+		}
+		name := util.SchemaTreeRoot(dir.Entry).Name
+		if cg.TrimOCPackage {
+			name = strings.TrimPrefix(name, "openconfig-")
+		}
+		name = strings.ReplaceAll(name, "-", "")
```

Comment:
rfc7950:
```
   identifier          = (ALPHA / "_")
                         *(ALPHA / DIGIT / "_" / "-" / ".")
```
Could you replace "_" and "." as well?

Reply:
https://go.dev/blog/package-names It seems we need to make the package names lowercase.

Reply:
switched and regex and toLower